src/metabase/lib/schema/ref.cljc

```diff
@@ -101,7 +101,7 @@
 (lib.hierarchy/derive :segment ::ref)
 
 (mbql-clause/define-tuple-mbql-clause :metric :- ::expression/type.unknown
-  #_metric-id [:schema [:ref ::id/metric]])
+  #_metric-id [:schema [:or [:ref ::id/metric] :string]])
```

Comment:
I enabled named references because of legacy usage. We have conversion tests that check if such references can be converted properly. I wonder if there is a better solution than this.